docs/modules/omevv_firmware_compliance_info.rst

```diff
@@ -0,0 +1,190 @@
+
+
+
+Parameters
+----------
+
+  clusters (optional, list, None)
+    The cluster details to retrieve the firmware compliance report.
+
+
+    cluster_name (True, str, None)
+      The cluster name of the hosts for which the firmware compliance report needs to be fetched.
+
+      If \ :emphasis:`servicetags`\  or \ :emphasis:`hosts`\  is provided, then firmware compliance report of only the specified hosts will be fetched and shown.
+
+
+    servicetags (optional, list, None)
+      The service tag of the hosts.
+
+      The hosts for which the firmware compliance reports needs to be fetched.
+
+
+    hosts (optional, list, None)
+      The IP address or hostname of the hosts.
+
+      The hosts for which the firmware compliance reports needs to be fetched.
+
+
+
+  hostname (True, str, None)
+    IP address or hostname of the OpenManage Enterprise Modular.
+
+
+  vcenter_username (False, str, None)
+    Username for OpenManage Enterprise Integration for VMware vCenter (OMEVV).
+
+    If the username is not provided, then the environment variable \ :envvar:`OMEVV\_VCENTER\_USERNAME`\  is used.
+
+    Example: export OMEVV\_VCENTER\_USERNAME=username
+
+
+  vcenter_password (False, str, None)
+    Password for OpenManage Enterprise Integration for VMware vCenter (OMEVV).
+
+    If the password is not provided, then the environment variable \ :envvar:`OMEVV\_VCENTER\_PASSWORD`\  is used.
+
+    Example: export OMEVV\_VCENTER\_PASSWORD=password
+
+
+  vcenter_uuid (False, str, None)
+    Universally Unique Identifier (UUID) of vCenter.
+
+    vCenter UUID details can be retrieved using \ :ref:`dellemc.openmanage.omevv\_vcenter\_info <ansible_collections.dellemc.openmanage.omevv_vcenter_info_module>`\  module.
+
+    If UUID is not provided, then the environment variable \ :envvar:`OMEVV\_VCENTER\_UUID`\  is used.
+
+    Example: export OMEVV\_VCENTER\_UUID=uuid
+
+
+  port (optional, int, 443)
+    OpenManage Enterprise HTTPS port.
+
+
+  validate_certs (optional, bool, True)
+    Whether to check SSL certificate. - If \ :literal:`true`\ , the SSL certificates will be validated. - If \ :literal:`false`\ , the SSL certificates will not be validated.
+
+
+  ca_path (optional, path, None)
+    The Privacy Enhanced Mail (PEM) file that contains a CA certificate to be used for the validation.
+
+
+  timeout (optional, int, 30)
+    The socket level timeout in seconds.
+
+
+
+
+
+Notes
+-----
+
+.. note::
+   - Run this module from a system that has direct access to Dell OpenManage Enterprise.
+
+
+
+
+Examples
+--------
+
+.. code-block:: yaml+jinja
+
+    
+    ---
+    - name: Fetch firmware compliance report of all the cluster
+      dellemc.openmanage.omevv_firmware_compliance_info:
+        hostname: "192.168.0.1"
+        vcenter_uuid: "xxxxx"
+        vcenter_username: "username"
+        vcenter_password: "password"
+        ca_path: "path/to/ca_file"
+
+    - name: Fetch firmware compliance report of all the hosts in the specific cluster
```

Comment:
- name: Retrieve a firmware compliance report of all the hosts in a specific cluster

Reply:
Done